src/components/article/ArticleComments.jsx

```diff
@@ -1,7 +1,76 @@
-import React from "react";
+import React, { useContext, useEffect, useState } from "react";
+import { NewsDataContext } from "../../data/NewData";
+import { BiSolidLike } from "react-icons/bi";
 
 const ArticleComments = () => {
-    return <div>ArticleComments</div>;
+    const [userAvatar, setUserAvatar] = useState("");
+    const [isMore, setIsMore] = useState(false);
+    const { activeArticle, userList } = useContext(NewsDataContext);
+
```

Comment:
Here as well article list should not need to be brought in from context and should instead be fetched from your api within this component.